apps/explorer/src/hooks/usePageView.ts

```diff
@@ -27,6 +27,6 @@ export function usePageView() {
             },
         });
 
-        postHog?.capture('$pageview', { url: pathname, name: network });
+        postHog?.capture('$pageview', { url: pathname, network });
```

Comment:
Re-naming this `network` for consistency and because it was an oversight in the original instrumentation PR. It'll also be a good opportunity to see how easy it is to manage changing-event data in PostHog